specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+        "in": "query",
+        "name": "api-version",
+        "description": "The requested API version.",
+        "required": true,
+        "type": "string"
+      }
+   },
+   "definitions": {
+      "Status": {
+         "description": "The status of the operation.",
+         "type": "string",
+         "enum": [
+            "NotStarted",
+            "Running",
+            "Failed",
+            "Canceled",
+            "Succeeded"
+         ]
+      },
+      "InnerError": {
+         "type": "object",
+         "description": "The inner error if a task in a batch failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Operation failed as a result of our safety system."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "ContentFilter",
+                  "Internal",
+                  "BlockedImages"
+               ],
+               "description": "The error code."
+            }
+         }
+      },
+      "Error": {
+         "type": "object",
+         "description": "The error if the request or operation failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Required query parameter 'api-version' is missing."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "BadRequest",
+                  "ContentFilter",
+                  "Internal",
+                  "TooManyRequests",
+                  "Unauthorized",
+                  "NotFound",
+                  "OperationNotInTerminalState",
+                  "BlockedImages",
+                  "RequestTimout"
+               ],
+               "description": "The error code."
+            },
+            "details": {
+               "type": "array",
+               "items": {
+                  "$ref": "#/definitions/InnerError"
+               }
+            }
+         },
+         "required": [
+            "message",
+            "code"
+         ]
+      },
+      "ErrorResponse": {
+         "type": "object",
+         "properties": {
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         },
+         "required": [
+            "error"
+         ]
+      },
+      "ImageResult": {
+         "type": "object",
+         "description": "The image url if successful, and an error otherwise.",
+         "properties": {
+            "url": {
+               "type": "string",
+               "description": "The image url.",
+               "example": "https://www.contoso.com"
+            },
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         }
+      },
+      "OperationResult": {
+         "type": "array",
```

Comment:
> "type": "array", [](http://example.com/codeflow?start=8&length=17)

is this really an array?

Reply:
Yes, the user can provide the number images that should be generated and therefore this can be an array.